skipruntime-ts/addon/README.md

```diff
@@ -0,0 +1,24 @@
+# @skipruntime/native
+
+This package provides a Node native add-on with bindings to the Skip runtime.
+_Either_ this package or its WebAssembly analogue
+[`@skipruntime/wasm`](https://www.npmjs.com/package/@skipruntime/wasm) is
+required to run a Skip reactive service.
+
+See the [docs](https://skiplabs.io/docs/api/server) for more details.
```

Comment:
Not sure where this should point. From docs/api/server people need to find the platform optional param of runService, which might be a bit of a stretch.

Reply:
Maybe just straight to the landing page like wasm?

Reply:
That might be best. If desired, not that e.g. http://localhost:3000/docs/api/server/functions/runService#parameters points to the param doc block, so might be doable too.

Reply:
I'll do the main landing page for now. It would be nice -- once we've hammered out the issues with native -- to just add a whole page to the docs site about the platform options and direct both these links there.